projects/go-lib/src/lib/components/go-input/go-input.component.spec.ts

```diff
@@ -33,4 +33,18 @@ describe('GoInputComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('ngOnInit()', () => {
+    it('sets default minlength if no minlength is specified', () => {
+      fixture.detectChanges();
+
+      expect(component.minlength).toBe(0);
+    });
+
+    it('sets default maxlength if no maxlength is specified', () => {
+      fixture.detectChanges();
+
+      expect(component.maxlength).toBe(524288);
+    });
+  });
```

Comment:
I was thinking we would want to test the logic in `ngOnInit` here rather than testing that the defaults gets set (I feel like testing the defaults is sort of just testing that Angular is working as intended, which we shouldn't have to do), so you could have:

```javascript
it('sets minlength to 0 if minlength is greater than maxlength', () => {
// ...
})

it('sets maxlength to 524288 if maxlength is greater than 524288', () => {
// ...
})
```

Reply:
Done and dusted!